scripts/sdk-coin-generator/template/README.md

```diff
@@ -0,0 +1,36 @@
+# BitGo sdk-coin-<%= symbol %>
+
+SDK coins provide a modular approach to a monolithic architecture. This and all BitGoJS SDK coins allow developers to use only the coins needed for a given project.
+
+## Installation
+
+All coins are loaded traditionally through the `bitgo` package. If you are using coins individually, you will be accessing the coin via the `@bitgo/sdk-api` package.
+
+In your project install both `@bitgo/sdk-api` and `@bitgo/sdk-coin-<%= symbol %>`.
+
+```shell
+npm i @bitgo/sdk-api @bitgo/sdk-coin-<%= symbol %>
+```
+
+Next, you will be able to initialize an instance of "bitgo" through `@bitgo/sdk-api` instead of via `bitgo`.
+
+```javascript
+import { BitGo } from '@bitgo/sk-api';
+
+async function init() {
+  const <%= symbol %> = await import('@bitgo/sdk-coin-<%= symbol %>');
+  BitGo.register(<%= symbol %>);
```

Comment:
I would expect this method to also take a coin identifier

Reply:
Good point, thanks.